src/ptyfwd/ptyfwd.c

```diff
@@ -0,0 +1,190 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <getopt.h>
+#include <unistd.h>
+
+#include "alloc-util.h"
+#include "build.h"
+#include "main-func.h"
+#include "pretty-print.h"
+#include "ptyfwd-util.h"
+#include "fd-util.h"
+#include "event-util.h"
+#include "strv.h"
+
+static bool arg_quiet = false;
+static char *arg_background = NULL;
+
+STATIC_DESTRUCTOR_REGISTER(arg_background, freep);
+
+static int help(int argc, char *argv[], void *userdata) {
+        _cleanup_free_ char *link = NULL;
+        int r;
+
+        r = terminal_urlify_man("systemd-pty-forward", "1", &link);
+        if (r < 0)
+                return log_oom();
+
+        printf("%1$s  [OPTIONS...] COMMAND ...\n"
+               "\n%5$sRun command with a custom terminal background color and title.%6$s\n"
```

Comment:
I assume the title is simply set to the command being executed? That is a bit unclear with the current wording